src/client/src/apps/SimpleLauncher/index.tsx

```diff
@@ -1,16 +1,74 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { ThemeProvider } from 'rt-theme'
 
 import { Launcher } from './Launcher'
+import { createServiceStub } from './spotlight/transport';
+import { getPlatformAsync, PlatformProvider } from 'rt-platforms';
+import { getFdc3 } from './spotlight/fdc3/fdc3';
+import { AutobahnConnectionProxy } from '../../rt-system';
+import { Fdc3Provider } from './spotlight/fdc3/context';
+import { TradeUpdatesProvider, PricingServiceProvider, ServiceStubProvider } from './spotlight/context';
+import BlotterService, { TradesUpdate } from '../MainRoute/widgets/blotter/blotterService';
+import { Observable, ReplaySubject } from 'rxjs';
+import PricingService from '../MainRoute/widgets/spotTile/epics/pricingService';
 
-export class SimpleLauncher extends React.Component {
-  render() {
-    return (
-      <ThemeProvider>
-        <Launcher {...this.props} />
-      </ThemeProvider>
-    )
+const autobahn = new AutobahnConnectionProxy(
+  process.env.REACT_APP_BROKER_HOST || location.hostname,
+  'com.weareadaptive.reactivetrader',
+  +(process.env.REACT_APP_BROKER_PORT || location.port),
+)
+
+export const SimpleLauncher: React.FC = () => {
+
+  const [platform, setPlatform] = useState()
+  const [fdc3, setFdc3] = useState()
+  const [serviceStub, setServiceStub] = useState()
+  const [pricingService, setPricingService] = useState<PricingService>()
+  const [tradesUpdates, setTradeUpdates] = useState<Observable<TradesUpdate>>()
```

Comment:
I'm wondering if this should be one large state object:
```
{
platform,
fdc3,
serviceStub,
pricingService,
tradesUpdates
}

then in the hook you only have to call `setState` once.
}

Reply:
done